docs/DeveloperGuide.md

```diff
@@ -685,17 +685,17 @@ testers are expected to do more *exploratory* testing.
 
 1. _{ more test cases …​ }_
 
-### Deleting a person
+### Deleting a event
```

Comment:
Maybe this should be `an` instead of `a`

Reply:
👍 I'll correct it